reference/conanfile/attributes.rst

```diff
@@ -970,6 +1016,10 @@ root folder of the package:
             # Get the sharedlinkflags property from OpenSSL package
             self.deps_cpp_info["openssl"].sharedlinkflags
 
+.. seealso::
+
+    Read more about :ref:`cpp_info_components_attributes_reference` in ``self.cpp_info``.
```

Comment:
This is linking to this same section 🤔  Probably we can remove this `seealso` section, right?

Reply:
you are right